### ISSUE TITLE
Permissions, some thoughts

### ISSUE THREAD

**Issue by MaxRek** [StrangeLoopGames/EcoSuggestions#1351]
Hello there

I was thinking, I was with a bunch of friends in a serv, where we share without limits our properties, crafts.

And I found that it was pretty long to always add everyone, one per one user. We were 7, and sure the search name was useful, but wouldn't be better to have something, like groups of friend ?
Like you register a group, and put the people you want in there, and the next time, everyone has permissions with only one add, and we're good to go. 

Also, another idea, is a "user permissions pre-config". 
It got to my mind that every time we're putting a cart, it was painful to add the right permissions, on only 4 days, we got like 6 problems like this, we were always removing cart when it's empty, so the rights was reset.

You get it, we set a config for new claims/carts/other vehicules ,so theirs permissions are set automatically.

I think that these problems I found are not really widespread, but people that fully work together like me, people who work together for a little bit of time, for like a contract, it would bring some "quality life" for them.

Thx u for the game

**Comment by danisani1** [comment]
Especially the second point is really a thing if you play together with someone. A pre-set would be really great for vehicles

**Comment by PixlRainbow** [comment]
you can give people access to deeds, and they should get permissions to all objects within that deed. If you place a cart within a claimed area, it might automatically be attached to the related deed, but I'm not sure.

**Comment by JestinJae** [comment]
My thought on this would be to just allow player-created groups, like the "Active", "Abandoned", "Admins", and "Everyone" groups.  This could also be a solution and would leverage something that is already implemented to some extent.

**Comment by PixlRainbow** [comment]
> like the "Active", "Abandoned", "Admins", and "Everyone" 
those are implemented as Demographics

**Comment by MaxRek** [comment]
Yeah, but so, there are this kind of groups where laws can't be applied to them, so the ones from society are still the best.

**Comment by 1337zorn** [comment]
this could be realised to add people who are invited within a groupchat

**Comment by PixlRainbow** [comment]
> I think that these problems I found are not really widespread, but people that fully work together like me, people who work together for a little bit of time, for like a contract

The thought just occured to me that this sounds a lot like a corporation.

> Corporation: *noun*
> A corporation is created when it is incorporated by a group of shareholders who share ownership of the corporation and pursue a common goal.
